my-webxr-app/src/App.tsx

```diff
@@ -15,22 +12,19 @@ import GraphingDataPointMenu from './components/GraphingDataPointMenu';
 import { PointSelectionProvider } from './contexts/PointSelectionContext';
 import './styles.css';
 import TestingOptions from './smoketest/TestingOptions';
-import DataPoint from './repository/DataPoint';
 import { rollbarConfig } from './utils/LoggingUtils';
 
 // minNum and maxNum will be from the csv file, just hardcoded for now
-const minNum: number = -10;
-const maxNum: number = 10;
+const maxNum: Array<number> = [10, 100, 1000];
```

Comment:
Do we want the graph to start with 10, 100, 1000? Or do we want them all to start the same?

Reply:
I should get rid of that. They were mean for visual confirmation, the max will be determined by the column data.